ArduCopter/mode.h

```diff
@@ -1218,7 +1220,7 @@ class ModeZigZag : public Mode {
     void auto_control();
     void manual_control();
     bool reached_destination();
-    bool calculate_next_dest(uint8_t position_num, Vector3f& next_dest) const;
+    bool calculate_next_dest(uint8_t position_num, bool use_wpnav_alt, Vector3f& next_dest, bool& terrain_alt) const;
```

Comment:
Might have been nice to use the `Location::AltFrame` enumeration here - extensive change, however.

Reply:
yes, that would be a big change so I will give that a miss..